### ISSUE TITLE
Mass Obsoletion: focus on branch 'disorder of orbital region' (MONDO:0002022)

### ISSUE THREAD

**Issue by sabrinatoro** [monarch-initiative/mondo#6744]
branch 'disorder of orbital region' (MONDO:0002022)

obsoletion candidate ID | obsoletion candidate label
-- | --
http://purl.obolibrary.org/obo/MONDO_0020196 | anomaly of the secretory and excretory apparatus of the lacrimal system
http://purl.obolibrary.org/obo/MONDO_0020147 | anophthalmia-microphthalmia syndrome
http://purl.obolibrary.org/obo/MONDO_0014753 | autosomal recessive optic atrophy
http://purl.obolibrary.org/obo/MONDO_0016647 | autosomal recessive Stickler syndrome
http://purl.obolibrary.org/obo/MONDO_0020205 | bulbar conjunctival dermoid or conjunctival dermolipoma
http://purl.obolibrary.org/obo/MONDO_0020163 | canthal anomaly
http://purl.obolibrary.org/obo/MONDO_0020194 | congenital alacrima
http://purl.obolibrary.org/obo/MONDO_0018888 | congenital cornea plana
http://purl.obolibrary.org/obo/MONDO_0018144 | congenital myasthenic syndromes with glycosylation defect
http://purl.obolibrary.org/obo/MONDO_0020375 | coralliform cataract
http://purl.obolibrary.org/obo/MONDO_0020219 | corneogoniodysgenesis
http://purl.obolibrary.org/obo/MONDO_0020145 | developmental defect of the eye
http://purl.obolibrary.org/obo/MONDO_0020197 | EEC syndrome and related syndrome
http://purl.obolibrary.org/obo/MONDO_0020164 | epicanthal fold
http://purl.obolibrary.org/obo/MONDO_0020195 | excretory apparatus of the lacrimal system anomaly
http://purl.obolibrary.org/obo/MONDO_0020155 | eyelid border anomaly
http://purl.obolibrary.org/obo/MONDO_0020158 | eyelids malposition disorder
http://purl.obolibrary.org/obo/MONDO_0020238 | inherited vitreous-retinal disease
http://purl.obolibrary.org/obo/MONDO_0035002 | isolated inherited retinal disorder
http://purl.obolibrary.org/obo/MONDO_0020237 | lens shape anomaly
http://purl.obolibrary.org/obo/MONDO_0020235 | lens size anomaly
http://purl.obolibrary.org/obo/MONDO_0020146 | major induction processes eye anomaly
http://purl.obolibrary.org/obo/MONDO_0020167 | malposition of external canthus
http://purl.obolibrary.org/obo/MONDO_0020181 | mesenchymatous palpebral tumor
http://purl.obolibrary.org/obo/MONDO_0020154 | microblepharon-ablephara syndrome
http://purl.obolibrary.org/obo/MONDO_0015368 | neuro-ophthalmological disease
http://purl.obolibrary.org/obo/MONDO_0019541 | non-infectious posterior uveitis
http://purl.obolibrary.org/obo/MONDO_0015217 | non-syndromic developmental defect of the eye
http://purl.obolibrary.org/obo/MONDO_0020275 | oculocutaneous or ocular albinism
http://purl.obolibrary.org/obo/MONDO_0020178 | palpebral lentiginosis
http://purl.obolibrary.org/obo/MONDO_0020180 | palpebral piliary tumor
http://purl.obolibrary.org/obo/MONDO_0020176 | palpebral sebaceous gland tumor
http://purl.obolibrary.org/obo/MONDO_0020203 | pigmented conjunctival lesion
http://purl.obolibrary.org/obo/MONDO_0020177 | pigmented palpebral tumor
http://purl.obolibrary.org/obo/MONDO_0020174 | precancerous lesion of palpebral epidermis
http://purl.obolibrary.org/obo/MONDO_0020345 | presynaptic congenital myasthenic syndrome
http://purl.obolibrary.org/obo/MONDO_0015485 | primary hereditary glaucoma
http://purl.obolibrary.org/obo/MONDO_0022407 | retinal ciliopathy due to mutation in Bardet-Biedl gene
http://purl.obolibrary.org/obo/MONDO_0022405 | retinal ciliopathy due to mutation in nephronophthisis gene
http://purl.obolibrary.org/obo/MONDO_0022397 | retinal ciliopathy due to mutation in the retinitis pigmentosa-1 gene
http://purl.obolibrary.org/obo/MONDO_0022399 | retinal ciliopathy due to mutation in the RPGR gene
http://purl.obolibrary.org/obo/MONDO_0022400 | retinal ciliopathy due to mutation in the RPGRIP gene
http://purl.obolibrary.org/obo/MONDO_0022404 | retinal ciliopathy due to mutation in Usher gene
http://purl.obolibrary.org/obo/MONDO_0020216 | secondary dysgenetic glaucoma
http://purl.obolibrary.org/obo/MONDO_0035075 | secondary early-onset glaucoma of genetic origin
http://purl.obolibrary.org/obo/MONDO_0020162 | secondary ectropion
http://purl.obolibrary.org/obo/MONDO_0020160 | secondary entropion
http://purl.obolibrary.org/obo/MONDO_0020193 | secretory apparatus of the lacrimal system anomaly
http://purl.obolibrary.org/obo/MONDO_0020346 | synaptic congenital myasthenic syndrome
http://purl.obolibrary.org/obo/MONDO_0020148 | syndromic aniridia
http://purl.obolibrary.org/obo/MONDO_0020156 | syndromic ankyloblepharon
http://purl.obolibrary.org/obo/MONDO_0020215 | syndromic corneal dystrophy
http://purl.obolibrary.org/obo/MONDO_0020165 | syndromic epicanthus
http://purl.obolibrary.org/obo/MONDO_0020210 | syndromic hyperopia
http://purl.obolibrary.org/obo/MONDO_0020211 | syndromic keratoconus
http://purl.obolibrary.org/obo/MONDO_0015310 | syndromic optic nerve hypoplasia
http://purl.obolibrary.org/obo/MONDO_0020157 | syndromic palpebral coloboma
http://purl.obolibrary.org/obo/MONDO_0020240 | syndromic retinitis pigmentosa
http://purl.obolibrary.org/obo/MONDO_0034954 | syndromic vitreoretinopathy

**Comment by nicolevasilevsky** [comment]
[spreadsheet](https://docs.google.com/spreadsheets/d/1FX5L4euFEcgQbCvxNx-buzBJbx3wa8sINpi9r0DZEHQ/edit#gid=1327031230)

**Comment by sabrinatoro** [comment]
@nicolevasilevsky I am reviewing your spreadsheet now. 
Please note that some "leaves the branch" are actually going to become "orphan". I made these in yellow.

**Comment by nicolevasilevsky** [comment]
@sabrinatoro this is ready for a final spot check - both orphaned and leaves the branch are done. Ready for mass obsoletion pipeline once #6772 is done

**Comment by sabrinatoro** [comment]
There are some comments from the community that should be reviewed, but this is future work. This issue can be closed for now.
